heaps/heap_sort.py

```diff
@@ -1,8 +1,19 @@
-
+from heaps.min_heap import MinHeap
 
 def heap_sort(list):
```

Comment:
👍 Since you are building a heap here, the space complexity is O(n) because you build a heap of n elements.